crates/admin/protobuf/cluster_ctrl_svc.proto

```diff
@@ -16,21 +16,23 @@ import "google/protobuf/empty.proto";
 package restate.cluster_ctrl;
 
 service ClusterCtrlSvc {
-  rpc GetClusterState(ClusterStateRequest) returns (ClusterStateResponse);
+  rpc GetClusterState(ClusterStateRequest) returns(ClusterStateResponse);
 
-  rpc ListLogs(ListLogsRequest) returns (ListLogsResponse);
+  rpc ListLogs(ListLogsRequest) returns(ListLogsResponse);
 
-  rpc DescribeLog(DescribeLogRequest) returns (DescribeLogResponse);
+  rpc DescribeLog(DescribeLogRequest) returns(DescribeLogResponse);
 
-  rpc ListNodes(ListNodesRequest) returns (ListNodesResponse);
+  rpc ListNodes(ListNodesRequest) returns(ListNodesResponse);
 
-  rpc TrimLog(TrimLogRequest) returns (google.protobuf.Empty);
+  rpc TrimLog(TrimLogRequest) returns(google.protobuf.Empty);
 
   rpc CreatePartitionSnapshot(CreatePartitionSnapshotRequest)
-      returns (CreatePartitionSnapshotResponse);
+      returns(CreatePartitionSnapshotResponse);
 
   rpc SealAndExtendChain(SealAndExtendChainRequest)
-      returns (SealAndExtendChainResponse);
+      returns(SealAndExtendChainResponse);
+
+  rpc FindTail(FindTailRequest) returns(FindTailResponse);
```

Comment:
[Minor] I think I liked it better before! 😅

Reply:
It's the formatter that did it 😅